### DISCUSSION TITLE
learning of a programming language using github

### DISCUSSION THREAD

**Discussion by open-source-god** [community/community#81599] [Accessibility]
### Select Topic Area

Question

### Body

How to effectively learn a programming language with the use of github?

**Comment by mojoece** [comment]
Contributing in the open source world will give you an idea that what are the requirements/knowledge you need to work in industries. Github is the right place for that.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I do want to note that I see that you have already asked a similar question in this category, and @mo-jo-dev answered that question as well. GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
